automatic/git.install/README.md

```diff
@@ -0,0 +1,23 @@
+# [<img src="" height="48" width="48" /> git.install](https://chocolatey.org/packages/git.install)
```

Comment:
@majkinetor you may wish to update the img src url here manually, if I remember correctly the icon url in the nuspec file is up to date, so this one won't be updated.